### PR TITLE
Add pytest-datadir recipe

### PR DESCRIPTION
**Pull request by nicoddemus**
cc @gabrielcnr @edisongustavo @gqmelo

### CODE REVIEW

File: recipes/pytest-datadir/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  requires:
+    - pytest
+  imports:
+    - pytest_datadir
+  commands:
+    - py.test --help
+
+about:
+  home: https://github.com/gabrielcnr/pytest-datadir
+  license: LGPL
```
Comment (ocefpaf) [inline recipes/pytest-datadir/meta.yaml]: [Here](https://github.com/gabrielcnr/pytest-datadir/blob/master/setup.py#L30) is says MIT. But the previous release seems to be `LGPL-3.0`.

Comment (nicoddemus) [inline recipes/pytest-datadir/meta.yaml]: Yep, just made a `0.2.0` release fixing this. 😄

File: recipes/pytest-datadir/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  home: https://github.com/gabrielcnr/pytest-datadir
+  license: MIT
+  summary: pytest plugin for manipulating test data directories and files.
+
+extra:
+  recipe-maintainers:
+    - nicoddemus
+    - gabrielcnr
+    - edisongustavo
+    - gqmelo
```
Comment (jakirkham) [inline recipes/pytest-datadir/meta.yaml]: Just to verify everyone is ok being a maintainer, quick poll of interest @gabrielcnr @edisongustavo @gqmelo ?

Comment (gqmelo) [inline recipes/pytest-datadir/meta.yaml]: Yes

File: recipes/pytest-datadir/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  requires:
+    - pytest
+  imports:
+    - pytest_datadir
+  commands:
+    - py.test --help
+
+about:
+  home: https://github.com/gabrielcnr/pytest-datadir
+  license: MIT
```
Comment (nicoddemus) [inline recipes/pytest-datadir/meta.yaml]: True. Created gabrielcnr/pytest-datadir#11.

**Pull request merged**